### PR TITLE
modified dockerfile to install bcftools

### PR DESCRIPTION
**Pull request by naqvia**
Modify Dockerfile to install bcftools

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,17 @@
 FROM rocker/tidyverse:4.2
 MAINTAINER naqvia@chop.edu
 WORKDIR /rocker-build/
+
+ENV BCF_VERSION 1.9
```
Comment (jharenza) [inline Dockerfile:5]: What is this doing? When load your image, it has version 1.13 - assuming it only installs latest.

>Program: bcftools (Tools for variant calling and manipulating VCFs and BCFs)
Version: 1.13 (using htslib 1.13+ds)

Can you try something like this instead: https://github.com/d3b-center/OpenPedCan-analysis/blob/74f73764ce56b3d580d825563e9b0a8709d734fe/Dockerfile#L69-L74

**Review by jharenza — CHANGES_REQUESTED** [review]
requesting installation of a specific version for reproducibility

**Comment by naqvia** [comment]
I used that example above and what we had on cavatica (bix tools). But it seems they do not working bc the paths do not exist anymore.  The latest Dockerfile works and has been built and pushed.

**Comment by naqvia** [comment]
I have tested it and it runs. Here are a few examples:
```
bcftools filter -i 'INFO/AF <=0.01' input/6836213c-dbe1-48b4-9a66-55dcc25ec024.single.vqsr.filtered.vep_105.vcf.gz
bcftools filter -i 'INFO/gnomad_3_1_1_AF_popmax <=0.01' input/6836213c-dbe1-48b4-9a66-55dcc25ec024.single.vqsr.filtered.vep_105.vcf.gz
bcftools filter -i 'INFO/DP >=100' input/6836213c-dbe1-48b4-9a66-55dcc25ec024.single.vqsr.filtered.vep_105.vcf.gz
```

These expressions need to be defined in the vcf header. Below is an example of when it fails I suggest this is done by the user and then the filtered vcf is ran through the tool.

**Review by jharenza — APPROVED** [review]

**Pull request merged**